src/routes/wallets.js

```diff
@@ -0,0 +1,19 @@
+var express = require('express'),
+    wallets = require('../api').wallets;
+
+
+module.exports = express.Router()
+  .post('/', function(req, res, next) {
+    var conf = req.app.get('conf');
+
+    // TODO validate request body
+    // TODO handle errors given back from api
+    wallets.create({
+        wallet_server: conf.wallet_server,
+        kdf_params: conf.kdf_params,
+        username: req.body.username,
+        password: req.body.password
+      })
+      .then(function() { res.send({}); })
```

Comment:
Ya, I've thought about this, but can't think of anything we could return that would be useful at the moment. Could we maybe keep it empty for now, and add fields when we need them?